spec/services/post_alerter_spec.rb

```diff
@@ -1102,6 +1102,27 @@ def set_topic_notification_level(user, topic, level_name)
       expect(JSON.parse(body)).to eq(payload)
 
     end
+
+    context "with push subscriptions" do
+      before do
+        Fabricate(:push_subscription, user: evil_trout)
+        SiteSetting.push_notification_time_window_mins = 10
```

Comment:
This is the default value, so not really necessary.

```suggestion
        SiteSetting.push_notification_time_window_mins = 10
```

Reply:
Since the specs here aren't directly testing the default value, I believe explicitly setting it here prevents the specs from failing in  the future if the default value ever changes